src/torque-offset-estimator.cpp

```diff
@@ -126,84 +125,83 @@ namespace dynamicgraph
           // Check the current iteration status
-          int _i = stdVecJointTorqueOffsets.size();
+          int i = current_progress;
           
-          if (_i < n_iterations) {
-            SEND_MSG("Collecting signals for iteration no:" + _i , MSG_TYPE_DEBUG_STREAM);
+          if (i < n_iterations) {
+            SEND_MSG("Collecting signals for iteration no:" + i , MSG_TYPE_DEBUG_STREAM);
 
             const Eigen::VectorXd& sot_enc = m_base6d_encodersSIN(iter);
             const Eigen::VectorXd& IMU_acc = m_accelerometerSIN(iter);
-            const Eigen::VectorXd& IMU_gyr = m_gyroscopeSIN(iter);
-            const Eigen::VectorXd& _tau = m_jointTorquesSIN(iter);
+            const Eigen::VectorXd& tau = m_jointTorquesSIN(iter);
 
-            Eigen::VectorXd _enc(m_model.nq);
-            base_sot_to_urdf(sot_enc.head<6>(), _enc.head<7>());
-            _enc.tail(m_model.njoints) = sot_enc.tail(m_model.njoints);
+            Eigen::VectorXd enc(m_model.nq);
+            enc.tail(m_model.nv-6) = sot_enc.tail(m_model.nv-6);
+            base_sot_to_urdf(sot_enc.head<6>(), enc.head<7>());
 
             //Get the transformation from ff(f) to torso (t) to IMU(i) frame:
             // fMi = oMf^-1 * fMt * tMi
-            se3::forwardKinematics(m_model,*m_data,_enc);
+            se3::forwardKinematics(m_model,*m_data,enc);
             se3::SE3 fMi = m_data->oMi[ffIndex].inverse()*m_data->oMi[torsoIndex]*m_torso_X_imu;
 
             //Move the IMU signal to the base frame.
             //angularAcceleration is zero. Intermediate frame acc and velocities are zero
-            const dynamicgraph::Vector _acc = fMi.rotation()*IMU_acc; //Torso Acceleration
+            const dynamicgraph::Vector acc = fMi.rotation()*IMU_acc; //Torso Acceleration
 
             //Deal with gravity predefined in robot model. Robot Z should be pointing upwards
-            m_model.gravity.linear() = _acc;
-
-            const Eigen::VectorXd& _tau_rnea = se3::rnea(m_model, *m_data, _enc,
-                                                         Eigen::VectorXd::Zero(m_model.njoints),
-                                                         Eigen::VectorXd::Zero(m_model.njoints));
-            const Eigen::VectorXd _current_offset = _tau - _tau_rnea.tail(m_model.njoints);
-            if(_current_offset.array().abs().maxCoeff() >=epsilon) {
-              SEND_MSG("Too high torque offset estimated for iteration"+ _i, MSG_TYPE_ERROR);
+            m_model.gravity.linear() = acc;
+            
+            //Set fixed for DEBUG
+            //m_model.gravity.linear() = m_model.gravity981;
+
+            const Eigen::VectorXd& tau_rnea = se3::rnea(m_model, *m_data, enc,
+                                                         Eigen::VectorXd::Zero(m_model.nv),
+                                                         Eigen::VectorXd::Zero(m_model.nv));
+            const Eigen::VectorXd current_offset = tau - tau_rnea;
```

Comment:
I wasn't sure that the size of the input sensor is N. Current is assuming that the size of the input sensor is N+6, same as rnea.
I'll change that